framework/db/Query.php

```diff
@@ -451,6 +461,7 @@ protected function queryScalar($selectExpression, $db)
             && empty($this->groupBy)
             && empty($this->having)
             && empty($this->union)
+            && empty($this->with)
```

Comment:
???

Reply:
What about this? property `$with` not exists in Query

Reply:
I removed this condition